flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/TaskManagerLocation.java

```diff
@@ -275,6 +275,15 @@ public static String getHostName(InetAddress inetAddress) {
         return hostName;
     }
 
+    /**
+     * Gets the location string of the TaskManager in the format of "$HOST:$PORT".
+     *
+     * @return The location string of the TaskManager.
+     */
+    public String getLocationString() {
```

Comment:
Could you help add some tests for this method?

Reply:
Sure